style.css

```diff
@@ -0,0 +1,107 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+
+/**
+*! Removing default browser style
+**/
+* {
+  box-sizing: border-box;
+}
+
+* body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+* a {
+  text-decoration: none;
+}
+
+* li {
+  list-style: none;
+}
+
+/**
+*! Header styles inputs here
+**/
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
```

Comment:
- [ ] Since the navbar contains both the 'logo' and the 'hamburger' menu I suggest adding `justify-content: space-between;` on the navbar class and by this the navbar will adapt to all different screen sizes and always ensures the existence of the space between the logo and the menu 💪.